zerver/lib/import_realm.py

```diff
@@ -245,25 +245,18 @@ def fix_upload_links(data: TableData, message_table: TableName) -> None:
 
 def fix_streams_can_remove_subscribers_group_column(data: TableData, realm: Realm) -> None:
     table = get_db_table(Stream)
-    admins_group = NamedUserGroup.objects.get(
-        name=SystemGroups.ADMINISTRATORS, realm=realm, is_system_group=True
-    )
     for stream in data[table]:
-        stream["can_remove_subscribers_group"] = admins_group
+        stream["can_remove_subscribers_group"] = get_stream_permission_default_group(
+            "can_remove_subscribers_group", realm
+        )
 
 
 def fix_streams_can_administer_channel_group_column(data: TableData, realm: Realm) -> None:
     table = get_db_table(Stream)
-    can_administer_channel_group_default_name = Stream.stream_permission_group_settings[
-        "can_administer_channel_group"
-    ].default_group_name
-    can_administer_channel_group_default_value = NamedUserGroup.objects.get(
-        name=can_administer_channel_group_default_name,
-        realm=realm,
-        is_system_group=True,
-    )
     for stream in data[table]:
-        stream["can_administer_channel_group"] = can_administer_channel_group_default_value
+        stream["can_administer_channel_group"] = get_stream_permission_default_group(
+            "can_administer_channel_group", realm
+        )
```

Comment:
Similarly these functions are probably a pattern we want to eliminate, and a good prep commit could be to just refactor these to be a loop over group settings, such that no changes are required to this file for new settings.